source/retryable-writes/retryable-writes.rst

```diff
@@ -52,65 +51,57 @@ Transaction ID
    The transaction ID identifies the transaction as part of which the command is
    running. In a write command where the client has requested retryable
    behavior, it is expressed by the top-level ``sessionId`` and ``txnNum``
-   fields. The ``sessionId`` component is the corresponding server session ID,
-   which is an opaque BSON value returned by the ``startSession`` command. The
-   ``txnNum`` component is a monotonically increasing (per server session),
+   fields. The ``sessionId`` component is the corresponding server session ID.
+   which is an opaque BSON value defined in the Driver Session specification.
+   The ``txnNum`` component is a monotonically increasing (per server session),
    positive 64-bit integer.
 
 DriverSession
-   Driver object representing a client session and the operations that can be
-   performed on it, as defined in the Driver Session specification. This object
-   is always associated with a server session. The name of this object MAY vary
-   across drivers.
-
-SessionOptions
-   Data structure defined in the Driver Session specification to provide options
-   when creating a DriverSession. This could be a strongly typed class in some
-   drivers, or it could be a loosely typed dictionary in other drivers.
+   Driver object representing a client session, which is defined in the Driver
+   Session specification. This object is always associated with a server
+   session; however, drivers will pool server sessions so that creating a
+   DriverSession will not always entail creation of a new server session. The
+   name of this object MAY vary across drivers.
 
 Additional terms may be defined in the Driver Session specification.
 
 Naming Deviations
 -----------------
 
-This specification defines the name for a new DriverSession option,
-``retryWrites``. To the extent possible, drivers SHOULD use the defined name.
-However, where a driver’s or language’s naming conventions would conflict,
-drivers SHOULD honor their existing conventions. For example, a driver may use
-``retry_writes`` instead of ``retryWrites``.
+This specification defines the name for a new MongoClient option,
+``retryWrites``. Drivers MUST use the defined name to ensure portability of
+connection strings across applications and drivers.
```

Comment:
Is MongoClientSettings what your driver uses for a dictionary of options taken by MongoClient in addition to the URI string? I assume those options are then merged with values from the URI query string in some fashion to produce the final options used for the MongoClient?

IIRC, the URI options are case-insensitive (unless that's just libmongoc). PHP happens to use the canonical, camel-case names of the URI options as our options dictionary keys, so I didn't stop to think that naming deviations would come up here. In any event, I'll reword this to allow flexibility for option names outside the URI string.